src/docfx/config/Config.cs

```diff
@@ -150,7 +150,7 @@ private static Config LoadCore(string docsetPath, string configPath, CommandLine
             try
             {
                 var optionConfigObject = options?.ToJObject();
-                var configObject = ExpandAndNormalize(JsonUtility.Merge(LoadConfigObject(docsetPath, configPath, extend, restoreMap), optionConfigObject));
+                var configObject = JsonUtility.Merge(LoadConfigObject(docsetPath, configPath, extend, restoreMap), optionConfigObject);
```

Comment:
>JsonUtility [](start = 35, length = 11)

expand again

Reply:
can we just expand optionConfigObejct?

---
In reply to: [210462579](https://github.com/dotnet/docfx/pull/3196#discussion_r210462579) [](ancestors = 210462579)